assets/css/styles.css

```diff
@@ -343,3 +515,6 @@
     top: 0px;
   }
 }
+

```

Comment:
Can you remove these unwanted lines @Neutx

Reply:
```suggestion
```

Reply:
Lines have been removed